news-bundle/src/Resources/contao/config/config.php

```diff
@@ -46,6 +46,7 @@
 $GLOBALS['TL_HOOKS']['generatePage'][] = array('contao_news.listener.generate_page', 'onGeneratePage');
 $GLOBALS['TL_HOOKS']['generateXmlFiles'][] = array('News', 'generateFeeds');
 $GLOBALS['TL_HOOKS']['replaceInsertTags'][] = array('contao_news.listener.insert_tags', 'onReplaceInsertTags');
+$GLOBALS["TL_HOOKS"]['generatePage'][] = array('News', 'setMetaData');
```

Comment:
We already have a listener for the `generatePage` hook three lines above. Wouldn't it make sense to re-use it?